src/pages/Task.tsx

```diff
@@ -197,6 +198,12 @@ const TaskCore: React.FC = () => {
         badge={
           <ErrorBoundary>
             <TaskStatusBadge status={status} blocked={blocked} />
+            {attributed && (
+              <>
+                <Space />
+                <TaskStatusBadge status={TaskStatus.Known} blocked={blocked} />
+              </>
```

Comment:
This can be rewritten to 
```
{attributed && 
<StyledTaskStatusBadge status={TaskStatus.Known} blocked={blocked} />
}
...
const StyledTaskStatusBadge = styled(TaskStatusBadge)`
margin-left: 10px;
`
```

Reply:
That's what I originally did and it didn't work. I'll try again.

Reply:
I tried it again. I can't explain why, but it doesn't work. 
![image](https://user-images.githubusercontent.com/13104717/104744083-734b1a00-571a-11eb-81cd-c87124d81840.png)